UPGRADING.md

```diff
@@ -1,6 +1,14 @@
 Upgrading SlackRubyBot
 ======================
 
+### Upgrading to >= 0.14.1
+
+#### Set up VCR explicitly
+
+Requiring `slack-ruby-bot/rspec` won't set up [VCR](https://rubygems.org/gems/vcr) anymore. If your spec suite implicitly relies on this you would need to set up VCR explicitly in your spec suite. Just follow standard VCR documentation.
```

Comment:
won't -> will no longer

Version should be 0.15.0, since it's a backwards incompatible change. Bump in version.rb, too, and CHANGELOG.